tests/cmd/auth/commands.go

```diff
@@ -70,6 +70,13 @@ func Whoami(user model.User) {
 	Eventually(sess).Should(Say("You are %s", user.Username))
 	Eventually(sess).Should(Exit(0))
 	Expect(err).NotTo(HaveOccurred())
+
+	// now test auth:whoami --all
+	sess, err := cmd.Start("deis auth:whoami --all", &user)
+	Eventually(sess).Should(Say("Username: %s", user.Username))
+	Eventually(sess).Should(Say("Email: %s", user.Email))
```

Comment:
First name and Last name are fields that cannot be edited with `deis auth:register`, but I have added the `IsStaff` field to check the extra field and I made sure that "First Name: " actually shows up in the output.